test/CrowdsaleGenTest.js

```diff
@@ -295,9 +298,9 @@ contract('LifCrowdsale Property-based test', function(accounts) {
 
     await runGeneratedCrowdsaleAndCommands({
       commands: [
-        {"type":"setWeiPerUSDinTGE","wei":0,"fromAccount":10},
-        {"type":"setWeiPerUSDinTGE","wei":0,"fromAccount":6},
-        {"type":"setWeiPerUSDinTGE","wei":3,"fromAccount":6}
+        { type:"setWeiPerUSDinTGE","wei":0,"fromAccount":10},
```

Comment:
why removing the double quotes to `type`, but not to the others? better not change any of them I guess :)

Reply:
I will remove the quotes on all properties.

Reply:
But try not to do it in this PR, it's already quite big

Reply:
ok, do you want me to remove all quotes changes too?

Reply:
I don't know, if it takes too much time just keep it, if you can easily revert just those changes that's better. Try not to introduce them the next time :)

Reply:
I will remove them when I re order the commits once its approved.